index.html

```diff
@@ -19,10 +21,12 @@ <h1>I’m Melkamu Microverse student, Glad to see you!</h1>
       <p>I’m a software developer! I can help you build a product feature or website Look through some of my 
           work and experience! If you like what you see and have a project you need coded don’t hesitate to contact me.</p>
       <a class="contact-label" href="#">LET’S CONNECT</a> 
-      <a class="my-address-links"href="#"><img src="./assets/m.png"></a>
-      <a class="my-address-links" href="#"><img src="./assets/git.png"></a>
-      <a class="my-address-links" href="#"><img src="./assets/t.png"></a>
-      <a class="my-address-links" href="#"><img src="./assets/lin.png"></a>
+      <ul>
+      <li class="my-address-list"><a class="my-address-links"href="#"><img src="./assets/m.png"></a> </li> 
+      <li class="my-address-list"> <a class="my-address-links" href="#"><img src="./assets/git.png"></a> </li> 
+      <li class="my-address-list"> <a class="my-address-links" href="#"><img src="./assets/t.png"></a> </li>
+      <li class="my-address-list">  <a class="my-address-links" href="#"><img src="./assets/lin.png"></a> </li>
+    </ul>
```

Comment:
- [ ] To ensure the functionality of your social icons and enable visitors to access your designated social domains, I highly recommend updating the links
-  replaced the href attribute values (#) with the appropriate URLs for each social platform. You need to replace your_twitter_username, your_instagram_username, and your_linkedin_username with your actual usernames or profile URLs on those platforms.